lib/Ed25519Signature2018.js

```diff
@@ -1,4 +1,37 @@
 /*!
- * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
  */
-export class Ed25519Signature2018 {}
+
+import {JwsLinkedDataSignature} from '@digitalbazaar/jws-linked-data-signature';
+import {
+  Ed25519VerificationKey2018
+} from '@digitalbazaar/ed25519-verification-key-2018';
+
+export class Ed25519Signature2018 extends JwsLinkedDataSignature {
+  /**
+   * @param type {string} Provided by subclass.
```

Comment:
need to setup jsdocs and fix jsdocs errors. I can make an issue to fix it or we can address it on this PR, whichever one works.

Reply:
@JSAssassin let's address em in this pr.

Reply:
Scratch that, I opened an issue for it - https://github.com/digitalbazaar/ed25519-signature-2018/issues/2